packages/expo/src/Linking/Linking.ts

```diff
@@ -129,7 +129,10 @@ function parse(url: string): ParsedURL {
     let expoPrefix: string | null = null;
     if (hostUriStripped) {
       const parts = hostUriStripped.split('/');
-      expoPrefix = `${parts.slice(1).join('/')}/--/`;
+      expoPrefix = `${parts
```

Comment:
don't need string interpolation anymore
```suggestion
      expoPrefix = parts
```